integration_tests/src/main/python/arithmetic_ops_test.py

```diff
@@ -297,10 +297,8 @@ def test_mod_pmod_by_zero(data_gen, overflow_exp):
 def test_cast_neg_to_decimal_err():
     # -12 cannot be represented as decimal(7,7)
     data_gen = _decimal_gen_7_7
-    exception_content = "Decimal(compact,-120000000,20,0}) cannot be represented as Decimal(7, 7)"
-    exception_str = "java.lang.ArithmeticException: " + exception_content if is_before_spark_330() \
-        and not is_databricks104_or_later() else "org.apache.spark.SparkArithmeticException: " \
-        + exception_content
+    exception_str = "java.lang.ArithmeticException: " if is_before_spark_330() \
```

Comment:
this doesn't really match what we have done for other tests, other tests we check the message but differentiate versions.  We were trying to match the exceptions as close as possible.

We also have https://github.com/NVIDIA/spark-rapids/issues/5196 which is to make exceptions match.

so I would rather see us go that direction

Reply:
Updated!